src/main/java/mate/academy/hibernate/relations/exceptions/DataProcessingException.java

```diff
@@ -0,0 +1,11 @@
+package mate.academy.hibernate.relations.exceptions;
+
+public class DataProcessingException extends RuntimeException {
+    public DataProcessingException(String message) {
+        super(message);
+    }
+
+    public DataProcessingException(String message, Throwable cause) {
```

Comment:
Am i missing something or you don`t use DataProcessingException with Throwable cause parameter?

Reply:
But should use